test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- Hey, in here also, the test.md file should not be visible in your GitHub repository. Try to make show it doesn't show in the repo.

Reply:
### README FILE

- [ ] Hey, please, kindly use this [readme template](https://github.com/microverseinc/readme-template), and customize it to suit the project needs to have a professional readme in your repository.

Reply:
## PR'S TITLE AND DESCRIPTION

![image](https://github.com/umarahgul/Hello-world/assets/59397159/6b20fad5-7fe0-4dc3-8c1e-1e94fc6c8541)



- Hey, great adding some title and description, however, we can improve on this to make it more descriptive. see few suggestions below:

**TITLE**

- This could be very simple as `Hello Microverse Milestone`

**DESCRIPTION**

- This also can start like this below:

**In this project milestone, I did the following:**

- Added the `index.html` file.
- Added the `style.css` file.
- Set up the `html-css` linter correctly.
- Ignored the `test.md` file in the `.gitignore` file.
- Added a professional readme file.
- Ensured there are no linters error.

> These are just a few things you can add to make your PR more descriptive. Please, kindly update them as you'd be doing this frequently.